autorel/changelog_generator/main/changelog_structures.py

```diff
@@ -11,6 +11,7 @@ class Contributor(object):
     def __init__(self, name, email, url):
         self._name = name
         self._email = email
+        self._url = url
```

Comment:
Please don't forget to split this commit at the end (this is not a rename-only commit).